### DISCUSSION TITLE
Add changing hotkeys in the desktop app

### DISCUSSION THREAD

**Discussion by vavakado** [element-hq/element-meta#2561] [Ideas]
#### What would you like to do?
I would like to have an option to rebind existing keymaps to another key combination because it would allow for better UX in my opinion. Also having a toggle to switch between a keymap being global(works even when alt-tabbed to another window) and local(works only when focused on element
#### Why would you like to do it?
Because current keymaps conflict with keymaps used in common applications(browsers, etc) and I would prefer let's say having my mute hotkey be the same in all VoIP applications.
#### How would you like to achieve it?
In the settings in the "Keyboard" tab I could press on a function(let's say "Toggle webcam on/off") and that would ask me to press a key combination and in return the function I pressed(in this case "Toggle webcam on/off") would be rebound to the key combination I pressed.

**Comment by kir-puch** [comment]
+1, its ridiculous how that's not a thing. Having a global keybind system like in Discord is so obvious
